src/py42/clients/_settings_managers.py

```diff
@@ -0,0 +1,520 @@
+from py42._internal.compat import str
+from py42._internal.compat import UserList
+from py42._internal.compat import UserDict
+from py42.exceptions import Py42Error
+
+
+def set_val(d, keys, value):
+    """Helper for setting nested values from a dict based on a list of keys."""
+    d = get_val(d, keys[:-1])
+    d[keys[-1]] = value
+
+
+def get_val(d, keys):
+    """Helper for getting nested values from a dict based on a list of keys."""
+    for key in keys:
+        d = d[key]
+    return d
+
+
+def bool_to_str(value):
+    if isinstance(value, bool):
+        return "true" if value else "false"
+    elif value in ("true", "false"):
+        return value
+    else:
+        raise ValueError("Value must be True or False")
+
+
+def str_to_bool(value):
+    if value == "true":
+        return True
+    if value == "false":
+        return False
+    else:
+        return value
+
+
+def days_to_minutes(days):
+    return str(int(days) * 1440)
```

Comment:
Is the expected behavior to always round floats down here?

```int(1.9)
>>> 1
```

Reply:
Might it make more sense to do `str(int(days * 1440.0))` or `str(int(round(days * 1440.0)))`

Reply:
Initially I was assuming the web UI only allowed you to enter whole digit days, but I guess you can enter floats. So I've updated it. Good catch.